Source/Scene/Cesium3DTileFeatureTableResources.js

```diff
@@ -0,0 +1,87 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/DeveloperError',
+        '../Renderer/WebGLConstants'
+    ], function(
+        defaultValue,
+        defined,
+        DeveloperError,
+        WebGLConstants) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    function Cesium3DTileFeatureTableResources(featureTableJSON, featureTableBinary) {
```

Comment:
Right now the approach of generating a JS array that holds onto the property's binary data just seems too expensive. I think it would be easier if this class stored a TypedArray view for each property, and then when getting an instance's property it just indexes into the typed array. To avoid allocations, the getter can take an optional `results` array parameter.

This is also make it easier to work with `Points3DTileContent` which wants TypedArrays for the color and position properties.